fenix/app/src/main/java/org/mozilla/fenix/browser/tabstrip/TabStrip.kt

```diff
@@ -66,6 +69,7 @@ private val tabStripIconSize = 24.dp
 private val spaceBetweenTabs = 4.dp
 private val tabStripStartPadding = 8.dp
 private val addTabIconSize = 20.dp
+private val longerTitleFadeWidth = 16.dp
```

Comment:
Could this just be `titleFadeWidth`? I am just wondering what the 'longer' really refers to. Are we only applying a fade is longer than 16dp? I think we are just saying the fade has a width of 16dp here, and it's always just there regardless of whether or not the title overflows.

Reply:
Yeah, makes sense to omit "longer" 👍